NextcloudTalk/BaseChatViewController.swift

```diff
@@ -964,6 +964,14 @@ import QuickLook
         NotificationPresenter.shared().present(text: NSLocalizedString("Message copied", comment: ""), dismissAfterDelay: 5.0, includedStyle: .dark)
     }
 
+    func didPressCopyLink(for message: NCChatMessage) {
+        let account = NCDatabaseManager.sharedInstance().activeAccount()
+        let url = "\(account.server)/call/\(room.token!)#message_\(message.messageId)"
```

Comment:
You need to take servers into account, that do not have pretty URLs enabled. See https://github.com/nextcloud/talk-ios/blob/ab0ee8e036864c349058c4954a956ed66ada10a5/NextcloudTalk/NCRoom.swift#L256-L269
(probably reusable as well)